Userland/DevTools/UserspaceEmulator/Emulator.cpp

```diff
@@ -272,15 +379,15 @@ const MmapRegion* Emulator::find_text_region(FlatPtr address)
     return matching_region;
 }
 
-String Emulator::create_backtrace_line(FlatPtr address)
+// FIXME: This interface isn't the nicest
+Optional<const MmapRegion*> Emulator::load_library_from_adress(FlatPtr address)
```

Comment:
Instead of returning `Optional<const MmapRegion*>`, let's just return `MmapRegion const*`